examples/guides/mp-restful-webservice/src/main/java/io/helidon/guides/mp/restfulwebservice/Main.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2018 Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2018,2019 Oracle and/or its affiliates. All rights reserved.
```

Comment:
* Copyright (c) 2018, 2019, Oracle and/or its affiliates. All rights reserved.

Minor correction - for consistency with the Oracle Legal-mandated syntax for the copyright notice.

Reply:
Done